File: 프로그래머스/lv2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    fibo = [0,1]
    if n == 0 :
        return 0
    elif n == 1 :
        return 1
    else:
        for i in range(n-1):
            fibo.append(fibo[-1] + fibo[-2])
        return fibo[-1]%1234567

File: 프로그래머스/2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    const fibo = Array(n+1).fill(0);
    fibo[1] = 1;
    for (let i = 2; i < n+1; i++) {
        fibo[i] = (fibo[i-2] + fibo[i-1]) % 1234567
    }
    return fibo[n];
}